### PR TITLE
Emelie Lindblom project Guess Who

### PR DESCRIPTION
**Pull request by waliem**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -26,10 +26,19 @@ <h1>Does the person have</h1>
...
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
```
Comment (hejmaria) [inline code/index.html:44]: no hat? Probably just missed it.

File: code/index.html [hunk 2]
```diff
@@ -52,12 +65,16 @@ <h1>Does the person have</h1>
       <div class="win-or-lose">
         <img
           class="guess-who-icon"
-          src="images/guess-who-bubble.png"
+          src="images/guesswhologo.png"
```
Comment (hejmaria) [inline code/index.html:68]: Nice! Looked better than the original.

File: code/index.html [hunk 3]
```diff
@@ -52,12 +65,16 @@ <h1>Does the person have</h1>
...
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
         <button id="playAgain" class="filled-button">PLAY AGAIN</button>
       </div>
+
+      <audio id="win" src="./win.wav" preload="auto"></audio>
```
Comment (hejmaria) [inline code/index.html:75]: really nice sound effects! Actually loved all of them.

File: code/script.js [hunk 4]
```diff
@@ -1,213 +1,218 @@
...
 // Array with all the characters, as objects
 const CHARACTERS = [
   {
-    name: 'Jabala',
-    img: 'images/jabala.svg',
-    hair: 'hidden',
-    eyes: 'hidden',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Jabala",
```
Comment (hejmaria) [inline code/script.js:14]: How come you changed all of the ' to " ?  Did you do that manually? Alot of work!

File: code/script.js [hunk 5]
```diff
@@ -218,108 +223,190 @@ const generateBoard = () => {
...
-  // What else should happen when we start the game?
+  charactersInPlay = CHARACTERS;
+  counterValue = 0;
+  updateDisplay(counterValue);
+  setSecret();
+  generateBoard();
+  winOrLose.style.display = "";
+};
+
+var counterVal = 0;
```
Comment (hejmaria) [inline code/script.js:248]: nice detail with the counter.

File: code/script.js [hunk 6]
```diff
@@ -218,108 +223,190 @@ const generateBoard = () => {
...
 
 // This function should be invoked when you click on 'Find Out' button.
 const checkQuestion = () => {
-  const { category, value } = currentQuestion
+  const { category, value } = currentQuestion;
 
+  if (category === "hair" || category === "eyes") {
+    if (secret[category] === value) {
+      filterCharacters(true);
+      console.log("Yes, correct!");
```
Comment (hejmaria) [inline code/script.js:279]: could remove the console logs if you wanted to shorten the code.

File: code/script.js [hunk 7]
```diff
@@ -218,108 +223,190 @@ const generateBoard = () => {
...
-start()
+start();
+generateBoard();
 
 // All the event listeners
-restartButton.addEventListener('click', start)
+restartButton.addEventListener("click", start);
+questions.addEventListener("change", selectQuestion);
+filter.addEventListener("click", checkQuestion);
+playAgainButton.addEventListener("click", start);
```
Comment (hejmaria) [inline code/script.js:412]: Great work Emelie!

**Review by hejmaria — APPROVED** [review]
Great work! You have met the requirements for the projects and I have played your game several times. It works perfectly. I missed the hat-option but I think you just missed to put it in the HTML. You have solved all the steps, and looking at your code I wonder why I even struggled that week. You make it look easy!

File: code/index.html (hunk 1 above)
Comment (waliem) [reply code/index.html:44]: oh yeah! I actually skipped it! I wanted to add but I focused on other things.

File: code/script.js (hunk 4 above)
Comment (waliem) [reply code/script.js:14]: haha I did not change them?! maybe some of my extensions did? lite the "prettier" or something? I def would not put myself through that 😄

File: code/script.js (hunk 6 above)
Comment (waliem) [reply code/script.js:279]: oh yes, thanks, that was missed!

File: code/script.js (hunk 7 above)
Comment (waliem) [reply code/script.js:412]: Thank you for your review!!

**Pull request closed without merging**